lib/Manager.php

```diff
@@ -241,9 +242,9 @@ public function loadLastCommentInfo(int $id): ?IComment {
 
 	public function resetAssignedSignalingServers(ICache $cache): void {
 		$query = $this->db->getQueryBuilder();
-		$query->select('*')
-			->from('talk_rooms')
-			->where($query->expr()->isNotNull('assigned_hpb'));
+		$query->select('r.*')
```

Comment:
did you miss replacing this one with the helper ?

Reply:
yes, although in this case it's not a problem as we only request against 1 DB. but could do for consistency